src/model_utils.py

```diff
@@ -2,6 +2,11 @@
 from fastai.vision import *
 from fastai.callbacks import *
 from fastai.vision.gan import *
+from PIL import Image, ImageDraw, ImageFont
```

Comment:
Plz rearrange imports with standard order. 
You can read this: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement